lobsterpy/cohp/describe.py

```diff
@@ -482,3 +482,106 @@ def write_description(self):
         """
         for textpart in self.text:
             print(textpart)
+
+    @staticmethod
+    def get_calc_quality_description(quality_dict):
+        """
+        This method will generate a text description of the LOBSTER calculation quality
+
+        Args:
+            quality_dict: python dictionary from lobsterpy.analysis.get_lobster_calc_quality_summary
+        """
+        text_des = []
+
+        for key, val in quality_dict.items():
+            if key == "minimal_basis":
+                if val:
+                    text_des.append("The LOBSTER calculation used minimal basis.")
+                if not val:
+                    text_des.append(
+                        "Consider rerunning the calculation with the minimum basis as well. Choosing a "
+                        "larger basis set is only recommended if you see a significant improvement of "
+                        "the charge spilling."
+                    )
+
+            elif key == "charge_spilling":
+                text_des.append(
+                    "The absolute and total charge spilling for the calculation are {} and {}, "

```

Comment:
We should have probably directly added % here.

Reply:
Yeah missed out there % symbol here. Will add it

Reply:
Done